src/drawer/__keys.ts

```diff
@@ -0,0 +1,30 @@
+const DIALOG_STATE_KEYS = [
```

Comment:
```suggestion
const DRAWER_STATE_KEYS = [
```

Rename them to Drawer everywhere